vizro-core/mkdocs.yml

```diff
@@ -17,8 +17,8 @@ nav:
           - Tables: pages/user_guides/table.md
           - Cards & Buttons: pages/user_guides/card_button.md
           - Containers: pages/user_guides/container.md
-
-      - CONTROLS:
+          - Tabs: pages/user_guides/tabs.md
```

Comment:
This is definitely the right place for the page because it's in `components` but I think we need to establish more cross linking between pages since as a user I might look for `Tabs` in a section next to `Layouts`.

Please can you add some references to this page from `Layouts` and from `Containers` and vice-versa? Ideally people should think of `Tabs` and `Containers` as just two different visual renderings of the same thing. And all this is ultimately just different ways to configure the layout of your page.